### PR TITLE
[#1191] add instance weighted loadbalancer

### PR DESCRIPTION
**Pull request by chengyouling**

### CODE REVIEW

File: spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/GovernanceHeaderStatusUtils.java [hunk 1]
```diff
@@ -17,13 +17,17 @@
...
+import java.util.Random;
+
 import org.springframework.core.env.Environment;
 
 public class GovernanceHeaderStatusUtils {
   private static final String RESPONSE_STATUS_HEADER_DEFAULT_KEY = "X-HTTP-STATUS-CODE";
 
   public static final String HEADER_KEY = "spring.cloud.servicecomb.governance.response.header.status.key";
 
+  public static final Random random = new Random();
```
Comment (liubao68) [inline spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/GovernanceHeaderStatusUtils.java:29]: Do not use global variable, and Random is not thread safe.

File: integration-tests/canary-serivcecomb-tests/servicecomb-canary-provider/src/main/resources/bootstrap.yaml [hunk 2]
```diff
@@ -37,6 +37,9 @@ spring:
         serverType: kie
         serverAddr: http://127.0.0.1:30110
         firstPullRequired: false
+      instance:
+        properties:
+          weight: 1
```
Comment (liubao68) [inline integration-tests/canary-serivcecomb-tests/servicecomb-canary-provider/src/main/resources/bootstrap.yaml:42]: WeightedServiceInstanceListSupplier  会使用这个配置项吗？

Comment (chengyouling) [reply integration-tests/canary-serivcecomb-tests/servicecomb-canary-provider/src/main/resources/bootstrap.yaml:42]: WeightedServiceInstanceListSupplier 中构建metadataWeightFunction中serviceInstance.getMetadata()会拿这个实例属性

File: spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/GovernanceHeaderStatusUtils.java (hunk 1 above)
Comment (chengyouling) [reply spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/GovernanceHeaderStatusUtils.java]: fixed

**Pull request closed without merging**